app/models/king.rb

```diff
@@ -1,7 +1,51 @@
 class King < Piece
+  def valid_move?(to_row, to_col)
```

Comment:
Yes. I had tried that since a regular king move is never obstructed since can only move to an adjacent square.